docs/modules/ROOT/pages/advanced-settings.adoc

```diff
@@ -0,0 +1,213 @@
+The default is `10`.
+* *couchbase.bulkIndexRetryWaitMs*: The number of milliseconds permitted to elapse between bulk-index retries.
+The default is `1000`.
+* *couchbase.ignoreFailures*: A value of `true` causes the plug-in to return a success-status to Couchbase Server, even if some documents could not be indexed.
+This prevents XDCR from stalling due to Elasticsearch indexing errors (for example, when a schema-change breaks some of the Elasticsearch type-mappings).
+The default is `false`.
+* *couchbase.ignoreDeletes*: Accepts as a value a list of one or more index-names, specified as a comma- or semicolon-delimited string.
+The plug-in ignores document-deletion and expiration for each named index.
+(This allows information to be indefinitely archived; and may therefore lead to significant growth in index-size.)
+* *couchbase.wrapCounters*: A value of `true` causes the plug-in to store all integer-values provided by Couchbase that are not valid JSON documents in a document whose format is `{＂value＂:<value>}`, and is stored under the Couchbase Server ID originally specified for the value.
+The default is `false`.
+* *couchbase.ignoreDotIndexes*: A value of `true` causes the plug-in to ignore indexes and aliases whose names have a period as their initial character: for `example`, `.kibana`, `.marvel`, `.clover`.
+The default is `true`.
+* *couchbase.includeIndexes*: Accepts as a value a list of one or more index-names, specified as a comma- or semicolon-delimited string.
+The plug-in ignores all indexes not included in the list.
+This prohibits creation of additional indexes for the storing of checkpoint metadata (potentially unnecessary, when the number of replicated indexes is small).
+Note that this setting takes precedence over *ignoreDotIndexes*, and therefore forces use of indexes whose names have a period as their initial character, even if their use has been explicitly prohibited by *ignoreDotIndexes* (see immediately above).
+* *couchbase.typeSelector*: Specifies the type selector class to be used for mapping documents to types.
+The following options are available:
+ ** `org.elasticsearch.transport.couchbase.capi.DefaultTypeSelector`: Maps all documents to the specified type (see immediately below).
+This is the default type selector, and so need not be explicitly specified.
+Type-specification is managed by the following settings:
+  *** *couchbase.typeSelector.defaultDocumentType*: The document type to which the `DefaultTypeSelector` maps all documents.
+The default is `couchbaseDocument`.
+  *** *couchbase.typeSelector.checkpointDocumentType*: The document type to which replication-checkpoint documents are mapped.
+The default is `couchbaseCheckpoint`.
+ ** `org.elasticsearch.transport.couchbase.capi.DelimiterTypeSelector`: If the document ID is in the format `<`_type_`><`_delimiter_`><*>`, documents are mapped to the type specified by `<`_type_`>`.
```

Comment:
When viewing the document on Github, this line doesn't render correctly. It shows up as:

    If the document ID is in the format <`type><`delimiter`><*>`,
    documents are mapped to the type specified by <`type>`. 

Does the same thing happen when the docs are processed by Antora, or does it look okay?